icone_arrow_boards/final_icone_report.py

```diff
@@ -16,7 +16,7 @@ def parse_xml_to_dict(xml_string):
 
 
 # IDS = ['13632527', '13632530', '13632531', '13632528']
-IDS = ['13632529']
+IDS = ["13632529"]
```

Comment:
Doesn't seem to be used?

Reply:
We're also limiting our route below (route_ids) to just I 70, is this the only route they will be deploying on? Should this be configurable?

Reply:
These are some scripts for generating icone reports, I have not taken the time to fully develop them, they are just tools I made. I would probably be more interested in removing them from this repo and moving them to a separate repo, if you don't think they belong here, than to fully flesh them out. They are used very rarely, and generate very specific results

Reply:
As mentioned offline, let's just add some comments in here and references from README so we know what these files are doing

Reply:
Just pushed an update the the README and added descriptions to each of the files!